File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/overload_subtype_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'overload_subtype'

include Overload_subtype

f = Foo.new
b = Bar.new

if spam(f) != 1
  raise RuntimeError, "foo"
end

if spam(b) != 2
  raise RuntimeError, "bar"
end

File: support_tools/swigwin-3.0.12/Examples/test-suite/go/overload_subtype_runme.go
package main

import . "./overload_subtype"

func main() {
	f := NewFoo()
	b := NewBar()

	if Spam(f) != 1 {
		panic("foo")
	}

	if Spam(b) != 2 {
		panic("bar")
	}
}
